typing/typedecl_variance.ml

```diff
@@ -333,7 +383,7 @@ let property : (prop, req) Typedecl_properties.property =
   let update_decl decl variance =
     { decl with type_variance = variance } in
   let check env id decl req =
-    if is_hash id then () else check_decl env decl req in
+    if is_hash id then () else check_decl env id decl req in
```

Comment:
In the code above (that GitHub would not le me point to):
```ocaml
let compute env decl req =
    compute_decl env ~check:false decl req in
```
the `compute` function ends up depending on an `Ident.t` parameter which is never used. By changing the definition of at least `compute_variance_decl` to not depend on the identifier in the case `check:false`, we can revert the changes to `Typedecl_properties` and frieds.